backend/danswer/connectors/slack/utils.py

```diff
@@ -1,6 +1,13 @@
+import time
+from collections.abc import Callable
 from typing import Any
 from typing import cast
 
+from slack_sdk.errors import SlackApiError
+from slack_sdk.web import SlackResponse
+
+_SLACK_LIMIT = 900
```

Comment:
Leave a comment for random numerical things, is this like messages per call or something?

Reply:
Added!